neo4j/error.go

```diff
@@ -94,3 +94,13 @@ func IsServiceUnavailable(err error) bool {
 	_, is = err.(*pool.PoolFull)
 	return is
 }
+
+// GetDatabaseErrorCode is a utility method to check if the provided error is DatabaseError and if so
+// it gets the error code.
+func GetDatabaseErrorCode(err error) string {
+	dbErr, is := err.(*db.DatabaseError)
+	if !is {
+		return ""
+	}
+	return dbErr.Code
+}
```

Comment:
add eol.

Reply:
Fixed it and signed the CLA but I have no idea why the check fails.

Reply:
Still CLA problems

Reply:
Is there anything I can do to resolve the CLA problem? I followed the process to send the email. I cause could be the first commit does not contains the name or email but the second has the correct github username so I don't know how the check performed.

Reply:
The build needs to rerun to check again and it triggers on commits.

We are very thankful for pull request and this one is nicely done but 1.8 is not the focus right now. Can you please have a look on 4.0-dev branch and see if the work done there solves your problem. 4.0 is on it's way, hopefully alpha pretty soon.